### PR TITLE
Add docs about how to deploy Scalar products from AWS Marketplace

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR adds the document about how to deploy Scalar products from AWS Marketplace.

In the AWS Marketplace, we can show an arbitrary URL to users on `Launch this software` page as follows.
AWS_Marketplace_Launch_this_software_page

I will add the URL of this document on the above page.
After that, users who subscribe to Scalar products can see how to deploy Scalar products (this document) from the AWS Marketplace page.

Please take a look!

### CODE REVIEW

File: docs/AwsMarketplaceGuide.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+   * [Scalar DL Ledger (BYOL)]() // TODO: Add URL of Marketplace page after it's published
+   * [Scalar DL Auditor (BYOL)]() // TODO: Add URL of Marketplace page after it's published
+
+1. Select **Continue to Subscribe**.
+
+1. Sign in to AWS Marketplace using your IAM user.  
+   If you have already signed in, this step will be skipped automatically.
+
+1. Read the **Terms and Conditions** and select **Accept Terms**.  
+   It takes some time. If it's done, you can see the current date in the **Effective date** column.
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:20]:
```suggestion
   It takes some time. When it's done, you can see the current date in the **Effective date** column.
```

File: docs/AwsMarketplaceGuide.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+   Also, you can see our product on the [Manage subscriptions](https://us-east-1.console.aws.amazon.com/marketplace/home#/subscriptions) page of AWS Console.
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:21]:
```suggestion
   Also, you can see our products on the [Manage subscriptions](https://us-east-1.console.aws.amazon.com/marketplace/home#/subscriptions) page of AWS Console.
```

File: docs/AwsMarketplaceGuide.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+## **[BYOL]** Deploy containers on EKS (Amazon Elastic Kubernetes Service) from AWS Marketplace using Scalar Helm Charts
+
+If you subscribe to Scalar products in AWS Marketplace, you can pull container images of Scalar products from the private container registry ([ECR](https://aws.amazon.com/ecr/)) of AWS Marketplace. This section explains how to deploy Scalar products on your [EKS](https://aws.amazon.com/eks/) Cluster from the private container registry of AWS Marketplace.
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:25]:
```suggestion
If you subscribe to Scalar products in AWS Marketplace, you can pull the container images of Scalar products from the private container registry ([ECR](https://aws.amazon.com/ecr/)) of AWS Marketplace. This section explains how to deploy Scalar products on your [EKS](https://aws.amazon.com/eks/) cluster from the private container registry.
```

File: docs/AwsMarketplaceGuide.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+             ```
+           * Scalar DL Schema Loader for Auditor (schema-loader-auditor-custom-values.yaml)
+             ```yaml
+             schemaLoading:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardl-schema-loader-auditor"
+                 version: "3.4.1"
+             ```
+
+1. Deploy the Scalar product using the Helm Chart with the above custom values file.
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:89]:
```suggestion
1. Deploy the Scalar products using the Helm Chart with the above custom values files.
```

File: docs/AwsMarketplaceGuide.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+   It takes some time. If it's done, you can see the current date in the **Effective date** column.
+   Also, you can see our product on the [Manage subscriptions](https://us-east-1.console.aws.amazon.com/marketplace/home#/subscriptions) page of AWS Console.
+
+## **[BYOL]** Deploy containers on EKS (Amazon Elastic Kubernetes Service) from AWS Marketplace using Scalar Helm Charts
+
+If you subscribe to Scalar products in AWS Marketplace, you can pull container images of Scalar products from the private container registry ([ECR](https://aws.amazon.com/ecr/)) of AWS Marketplace. This section explains how to deploy Scalar products on your [EKS](https://aws.amazon.com/eks/) Cluster from the private container registry of AWS Marketplace.
+
+1. Update the custom values file of the Helm Chart of a Scalar product you want to install.  
+   You need to specify the private container registry (ECR) of AWS Marketplace and the version (tag) as the value of `[].image.repository` and `[].image.version (tag)` in the custom values file.  
+   * Examples
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:29]: It looks too nested, so can you remove `Examples` and make it something like this?

* Scalar DB examples
* Scalar DL examples

File: docs/AwsMarketplaceGuide.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+           * Scalar DL Schema Loader for Auditor (schema-loader-auditor-custom-values.yaml)
+             ```yaml
+             schemaLoading:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardl-schema-loader-auditor"
+                 version: "3.4.1"
+             ```
+
+1. Deploy the Scalar product using the Helm Chart with the above custom values file.
+   * Examples
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:90]: Let's use the same structure as above for clarity.
(The above custom values files are kind of grouped into Scalar DB and Scalar DL, so let's make this also look the same)

File: docs/AwsMarketplaceGuide.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+             ```yaml
+             schemaLoading:
+               image:
+                 repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardl-schema-loader-auditor"
+                 version: "3.4.1"
+               imagePullSecrets:
+                 - name: "reg-ecr-mp-secrets"
+             ```
+
+1. Deploy the Scalar product using the Helm Chart with the above custom values file.
```
Comment (feeblefakie) [inline docs/AwsMarketplaceGuide.md:205]:
```suggestion
1. Deploy the Scalar products using the Helm Chart with the above custom values files.
```

**Review by feeblefakie — COMMENTED** [review]
Overall, looking good! Thank you!
Left minor suggestions. PTAL!

File: docs/AwsMarketplaceGuide.md (hunk 5 above)
Comment (kota2and3kan) [reply docs/AwsMarketplaceGuide.md]: I updated it in 22d1432c1d0b1c987d47c4aa477e495982b749ed.

File: docs/AwsMarketplaceGuide.md (hunk 6 above)
Comment (kota2and3kan) [reply docs/AwsMarketplaceGuide.md]: I updated it in 22d1432c1d0b1c987d47c4aa477e495982b749ed.

**Comment by kota2and3kan** [comment]
@feeblefakie  
Thank you for your review!

I updated the suggested things.
Please take a look again!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**